Applets/UI/Modifiers/FilledCellModifier.swift

```diff
@@ -14,16 +14,16 @@ struct FilledCellModifier: ViewModifier {
         content
             .background(
                 GeometryReader { geometry in
-                    ZStack {
+//                    ZStack {
```

Comment:
Do not commit commented code, please. Remove it before commit and pull request, otherwise our code will become messy at some point